commerce-test/src/testIntegration/java/com/liferay/commerce/order/stock/test/OrderStockManagementTest.java

```diff
@@ -123,16 +148,23 @@ public void testProductAvailability() throws Exception {
 		);
 
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_company.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addB2CCommerceOrder(
-			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
+			_company.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId(),
+			_commerceChannel.getSiteGroupId());
+
+		_commerceOrders.add(commerceOrder);
 
-		CPInstance cpInstance = CPTestUtil.addCPInstance(_group.getGroupId());
+		CPInstance cpInstance = CPTestUtil.addCPInstance(_company.getGroupId());
```

Comment:
why _company groupId instances should be related to catalog no global group

Reply:
The company groupId is used to get the service context from which we take the user and so the catalogs.